src/service/chain_rewardable_entities.proto

```diff
@@ -24,30 +21,36 @@ message pub_key {
 enum mobile_hotspot_device_type {
   mobile_hotspot_device_type_UNKNOWN =
       0; // Unknown device type, should not be used
-  mobile_hotspot_device_type_WIFI_INDOOR = 1;
-  mobile_hotspot_device_type_WIFI_OUTDOOR = 2;
-  mobile_hotspot_device_type_WIFI_DATA_ONLY = 3;
+  mobile_hotspot_device_type_CBRS = 1;
+  mobile_hotspot_device_type_WIFI_INDOOR = 2;
+  mobile_hotspot_device_type_WIFI_OUTDOOR = 3;
+  mobile_hotspot_device_type_WIFI_DATA_ONLY = 4;
 }
 
 message mobile_hotspot_metadata {
-  // Timestamp when the NFT was created
-  uint64 created_at_ms = 1;
   // Serial number of the mobile hotspot device (can be null or empty... in the
   // case of data-only devices)
   optional string serial_number = 2;
   // Type of the mobile hotspot device
   mobile_hotspot_device_type device_type = 3;
   // The asserted location in hex format
-  string asserted_hex = 4;
+  optional string asserted_hex = 4;
+  optional uint32 antenna = 5;
+  optional int32 elevation = 6;
+  optional uint32 azimuth = 7;
+  optional uint32 mechanical_down_tilt = 8;
+  optional uint32 electrical_down_tilt = 9;
+  bool is_full_hotspot = 10;
```

Comment:
what does `is_full_hotspot` mean? 

Are these fields optional for the case where an NFT exists, but a hotspot has not yet been asserted? 

We might consider NOT emitting events until the hotspot has been asserted.

Reply:
You can actually assert a hotspot without this information. Deployment info is optional.

is_full_hotspot is how we denote data only hotspots. Basically if it's eligible for PoC. We can rename the field, but this is what we use everywhere else iirc

Reply:
is_full_hotspot is a bit cryptic and sounds like its already covered by `device_type`. If it overlaps with device_type, let's remove it.